### PR TITLE
Guice-based Beadledom clients will now use the correct time unit for time outs

### PR DESCRIPTION
**Pull request by charanpanchagnula**
### What was changed? Why is this necessary?

> Addresses issue https://github.com/cerner/beadledom/issues/144

### How was it tested?

> White box test and integration test was updated

### How to test

> This is bare minimum acceptable testing

- [ ] `./mvnw clean install -U`

### CODE REVIEW

File: client/beadledom-client-example/example-client/src/main/java/com/cerner/beadledom/client/example/client/ExampleOneClientModule.java [hunk 1]
```diff
@@ -54,9 +54,9 @@ PaginatedClientResource providePaginatedResource(
   BeadledomClientConfiguration provideClientConfig() {
     return BeadledomClientConfiguration.builder()
         .maxPooledPerRouteSize(60)
-        .socketTimeoutMillis(60)
-        .connectionTimeoutMillis(60)
-        .ttlMillis(60)
+        .socketTimeoutMillis(200)
```
Comment (charanpanchagnula) [inline client/beadledom-client-example/example-client/src/main/java/com/cerner/beadledom/client/example/client/ExampleOneClientModule.java]: I was getting read timeouts for integration tests in ClientServiceSpec so i bumped this up a little bit. Let me know if this is a big mistake

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 ### Fixed
 * JAX-RS providers with binding annotations will no longer be registered in the resteasy provider factory for the server.
+* Guice-based Beadledom clients will now use the correct time unit for time outs.
```
Comment (PrabhatShankar2102) [inline CHANGELOG.md]: nit: this should be "timeouts".

Comment (charanpanchagnula) [reply CHANGELOG.md]: why? time out is not a single word :S

Comment (johnlcox) [reply CHANGELOG.md]: I think it is a single word in programming at least.

Comment (charanpanchagnula) [reply CHANGELOG.md]: i changed it to time-out: https://github.com/cerner/beadledom/pull/147/commits/f6fb6fc346add9772d79249dfe8bad813ce7f2b6

Reference: https://en.wikipedia.org/wiki/Timeout_(computing)

**Pull request merged**